dump-extensions/openpower-dumps/resource_dump_entry.cpp

```diff
@@ -61,13 +61,26 @@ void Entry::delete_()
 {
     auto srcDumpID = sourceDumpId();
     auto dumpId = id;
+
+    if ((offloadUri() != "") && (phosphor::dump::isHostRunning()))
```

Comment:
offloadUri() != ""
can we do offloadUri.empth()

Reply:
may be we can watch on hoststate property store it internally rather than making the D-Bus call every time.

https://github.com/ibm-openbmc/powervm-handler/blob/main/dump/host_state_watch.cpp

may be u can create an issue which we can work on later

Reply:
No that is overhead... this call comes only once in a while in the field